### PR TITLE
Condense yaml

### PR DESCRIPTION
**Pull request by thl024**
Condenses minimal.yaml into around 50-60 lines from 100 lines. 

**Important**: Other yamls are broken right now because they do not conform to the new syntax for yaml key class (class is classpath + class now)

### CODE REVIEW

File: configurations/example.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # File metadata, to be printed by the simulator
 name:
-  Random Cases, Preset Bases, Preset Travel Times
+  "Random Cases, Preset Bases, Preset Travel Times"
```
Comment (hansyuan) [inline configurations/example.yaml]: Is there a reason why these are in a string?

File: configurations/minimal.yaml [hunk 2]
```diff
@@ -1,97 +1,61 @@
...
-  classpath: ems.generators.location.circle
+bases: # Initializes 6 bases in the UCSD area
+  class: ems.datasets.location.location_set.LocationSet
+  latitudes: [32.877328, 32.873803, 32.876727, 32.875811, 32.884714, 32.888589]
+  longitudes: [-117.234695, -117.239527, -117.242650, -117.230869, -117.236528, -117.242113]
 
-  # Defines coordinates for the center of the circle
+ambulances: # Initializes ambulances; one stationed at each of the 6 bases
+  class: ems.datasets.ambulance.custom_ambulance_set.CustomAmbulanceSet
+  bases: "@bases"
```
Comment (hansyuan) [inline configurations/minimal.yaml]: Would it make more sense to name the first order object differently from the reference? For example: 

`bases: @bases_set`

File: configurations/simple.yaml [hunk 3]
```diff
@@ -11,7 +11,7 @@ simulator:
 
   # Defines the ambulances to be used in the simulation
   ambulances:
-    class: CustomAmbulanceSet
+    class: BaseSelectedAmbulanceSet
```
Comment (hansyuan) [inline configurations/simple.yaml]: Will this code work? If so, we have two different ways of writing the same potentially user-confusing thing

File: configurations/minimal.yaml (hunk 2 above)
Comment (hansyuan) [reply configurations/minimal.yaml]: Another question: Will `@ base_set` work?

File: configurations/simple.yaml (hunk 3 above)
Comment (hansyuan) [reply configurations/simple.yaml]: I'm referring to the presence of both `class` and `(class, classpath)`

File: configurations/example.yaml (hunk 1 above)
Comment (thl024) [reply configurations/example.yaml]: YAML does not accept @ symbols (which we needed for referencing objects) unless it's explicitly a string. 

e.g. @object_tag must change to "@object_tag"

So I went ahead and added quotations to all strings.

File: configurations/minimal.yaml (hunk 2 above)
Comment (thl024) [reply configurations/minimal.yaml]: Hmmm I feel like it would be better to keep the naming the same

File: configurations/simple.yaml (hunk 3 above)
Comment (thl024) [reply configurations/simple.yaml]: No that code doesn't work. I didn't modify the other yamls yet.

File: configurations/minimal.yaml (hunk 2 above)
Comment (hansyuan) [reply configurations/minimal.yaml]: Well, if we explain `@` clearly, it should be no problem.

File: configurations/simple.yaml (hunk 3 above)
Comment (hansyuan) [reply configurations/simple.yaml]: Okay, np.

**Pull request merged**